test/python/test_tf2ngraph_functions.py

```diff
@@ -58,35 +58,3 @@ def test_config_updater_api(self):
             'ngraph_backend': 'CPU',
             'device_id': '0'
         }
-
-    # In this test we test that the spaces are handled correctly
-    @pytest.mark.parametrize(('ng_device',),
-                             (('{abc:1,def:2}',), ('{abc:1, def:2}',),
-                              ('{abc :1,def: 2}',), ('{abc:1,def: 2} ',)))
-    def parse_extra_params_string(self, raw_string):
```

Comment:
removing these chunk of tests, sicne now we do not string parse, instead we use the json library